Formula/lib/libimobiledevice.rb

```diff
@@ -16,11 +16,12 @@ class Libimobiledevice < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "c418aa763659463002269201586cd356c69e6a233b2040ced02a352498b362d8"
   end
 
-  # libimobiledevice-glue is required for building future versions
+  # libimobiledevice-glue and libtatsu are required for building future versions
   # Move outside of HEAD clause when there's a new release.
   head do
     url "https://github.com/libimobiledevice/libimobiledevice.git", branch: "master"
     depends_on "libimobiledevice-glue"
+    depends_on "libtatsu"
   end
```

Comment:
this can be done as separate/followup pr so that it wont trigger a new bottle publishing

Reply:
I can create a new PR with this commit only, however this change is required and should be merged right after this PR as current `HEAD` builds will already require `libtatsu` (like it did with `libimobiledevice-glue` when it has been introduced) and `libimobiledevice` hasn't had a release in 4 years and it's not expected to get one anytime soon.

If this then makes no difference as changes in the Formula would automatically result in a bottle publishing, I'd rather prefer leaving this change inside this PR.